content/blog/esc-projects-environment-tags-launch/index.md

```diff
@@ -63,6 +63,13 @@ We offer numerous ways for you to get started with Projects via the Pulumi Web C
   pulumi/cloud-pe/demo
  ```
 
+### Breaking Changes
+
+All existing environments should continue to work with the introduction of Projects, belonging now to a `default` project. However, when moving existing environments out of the default project you may notice some differences that we have outlined below to be proactive in communicating possible actions you'll need to migrate your environments.
+
+* If your environment was interpolating the `context.currentEnvironment.name` or `context.rootEnvironment.name` from [Contextual information](/docs/esc/environments/#pulumi-contextual-information) this value will now include the project for any environments outside of the `default` project.
```

Comment:
Can we add an example here?

Reply:
I link to the docs which I think should be enough